manifests/init.pp

```diff
@@ -51,14 +51,17 @@
     ensure => $package_ensure,
   }
 
+   $ensure_status = $enable ? { 0 => 'absent',  default  => 'present' }
```

Comment:
Maybe a case for https://github.com/puppetlabs/puppetlabs-stdlib/blob/main/REFERENCE.md#stdlib--ensure